### PR TITLE
Install NodeJS and npm with the core helper

### PR DESCRIPTION
**Pull request by tituspijean**
Since the app fell to level 1, I noticed all the warnings about NodeJS version, and it looks like it is not install the "proper" way, with the helper. Let's try that.

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by tituspijean** [comment]
!testme

**Comment by mhfowler** [comment]
Thanks for trying to fix this! I'm currently looking into this now. I think there is a way to tell ArchiveBox where to look for the node binary it needs.

**Comment by mhfowler** [comment]
!testme

**Comment by mhfowler** [comment]
!testme

**Comment by tituspijean** [comment]
> Thanks for trying to fix this! I'm currently looking into this now. I think there is a way to tell ArchiveBox where to look for the node binary it needs.

Ah nice! I concede that I tried that a bit blindly, it's nice to see they at least offer a setting.

🤞

File: scripts/install [hunk 1]
```diff
@@ -143,6 +143,7 @@ cd $final_path; ynh_npm ci
 
 ynh_node_path=$ynh_node
 ynh_script_progression --message="ynh_node_path: $ynh_node_path" --weight=1
+ynh_script_progression --message="ynh_node_load_PATH: $ynh_node_load_PATH" --weight=1
```
Comment (tituspijean) [inline scripts/install:146]: Actually, why do to add it as a text displayed to the user? It seems unnecessary.

Comment (mhfowler) [reply scripts/install:146]: @tituspijean yup just a debug message. I still haven't quite got this to work. will see on this next test run if this fixes the problem.

**Comment by mhfowler** [comment]
Cool, it looks like your fix, plus also using ynh_node_load_PATH for archivebox commands fixed the issue. There is another issue in the upgrade part I will check out now.

**Comment by mhfowler** [comment]
seeing this error in the logs, 
```npm ERR! fatal: write error: Disk quota exceeded```
wonder if thats something about CI running out of space

**Comment by mhfowler** [comment]
!testme

**Pull request merged**